### PR TITLE
Grafana stats view as mentioned in #3812

### PR DESCRIPTION
**Pull request by utkarshcmu**
@torkelo / @bergquist Please review / merge. For screenshots check #3812

### CODE REVIEW

**Comment by bergquist** [comment]
Seems nice! Two minor changes needed before merging to master
adminStatCtrl should be typescript.

File: pkg/services/sqlstore/stats.go [hunk 1]
```diff
@@ -46,3 +47,53 @@ func GetSystemStats(query *m.GetSystemStatsQuery) error {
...
+      (
+        SELECT COUNT(*)
+        FROM ` + dialect.Quote("playlist") + `
+      ) AS playlist_count,
+      (
+        SELECT COUNT (DISTINCT ` + dialect.Quote("dashboard_id") + ` )
+        FROM ` + dialect.Quote("star") + `
+      ) AS starred_db_count,
+      (
+        SELECT SUM ( ` + dialect.Quote("is_admin") + ` )
```
Comment (bergquist) [inline pkg/services/sqlstore/stats.go]: I think the best solution for counting grafana admins would be count(*) where is_admin. Makes it more obvious what the query is about.

**Comment by utkarshcmu** [comment]
@torkelo/@bergquist Made the suggested changes. Please review/merge.

**Comment by bergquist** [comment]
great work!

**Comment by utkarshcmu** [comment]
Thanks for merging this PR, this will be so helpful for admins to get a quick status of grafana in just 2 clicks.

**Pull request merged**